.circleci/config.yml

```diff
@@ -1,107 +1,46 @@
 version: 2.1
 executors:
-  pw-focal-development:
+  linux:
```

Comment:
it might just be faster to revert all changes to this file. Looks like it's getting a few things from master from 1.8.1

Reply:
Whoa, strange. I could've sworn I copied in whatever was on master. All fixed now.